glsl/combined.frag

```diff
@@ -227,8 +229,8 @@ void main()
 	}
 	else
 	#if __VERSION__ == 120
-		gl_FragColor = c;
+		gl_FragColor = c * vTint;
 		#else
-		fragColor = c;
+		fragColor = c * vTint;
 		#endif
```

Comment:
While we're here: Can we fix the indentation? (And possibly add curly braces.)

Reply:
These are GLSL preprocessor commands, not runtime code!